Formula/tmux.rb

```diff
@@ -22,6 +23,7 @@ class Tmux < Formula
   depends_on "pkg-config" => :build
   depends_on "libevent"
   depends_on "ncurses"
+  depends_on "utf8proc"
```

Comment:
Hey @j16sdiz, thanks for this PR ❤️
I was wondering if `utf8proc` is always needed as a dependency or just when building from source?
If that's the case we should probably mark it as such: `depends_on "utf8proc" => :build`
Let me know what you think :wink:
Stay awesome! :muscle: